Dockerfile

```diff
@@ -23,4 +23,7 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     yarn install && \
     yarn build
 
+# The "nobody" user, represented numerically for optimal compatibility with Kubernetes security policies
+USER 65534
```

Comment:
@funkypenguin Thanks for the PR! Does this user always exist? Can we create a non-root user and use it?

Reply:
Hi @harupy! Yes, `nobody` always exists in the `python:3.8` image, but there's no reason we couldn't create a dedicated, non-root user in Dockerfile, and use that instead :)

Incidentally (sorry to hijack this thread), is it intentional that boto3 is _not_ available in the official mflow Docker image (noticed today when trying to do S3-based artifacts)?

Cheers!
D

Reply:
@funkypenguin I'm reading https://askubuntu.com/questions/329714/what-is-the-purpose-of-the-nobody-user to understand what `nobody` is for.

Reply:
According to https://wiki.ubuntu.com/nobody, we should not use `nobody`.

Reply:
> is it intentional that boto3 is not available in the official mflow Docker image (noticed today when trying to do S3-based artifacts)?

No, it's not. When we released [the official mlflow Docker image](https://github.com/mlflow/mlflow/pkgs/container/mlflow), we weren't sure which optional packages (such as boto3) the image should include so it just includes `mlflow`. We can consider adding more packages if there is a request.

Reply:
> According to [wiki.ubuntu.com/nobody](https://wiki.ubuntu.com/nobody), we should not use `nobody`.

Ah, TIL. I'll update the PR with a dedicated user instead!

Reply:
Updated with a dedicated user :)